pages/linux/journalctl.md

```diff
@@ -3,17 +3,17 @@
 > Query the systemd journal.
 > More information: <https://manned.org/journalctl>.
 
-- Show all messages from this [b]oot:
+- Show all messages with priority level 3 (errors) from this [b]oot:
 
-`journalctl -b`
+`journalctl -b --priority={{3}}`
 
 - Show all messages from last [b]oot:
 
 `journalctl -b -1`
 
-- Show all messages with priority level 3 (errors) from this [b]oot:
+- Delete journal logs which are older than 2 days:
 
-`journalctl -b --priority={{3}}`
+`journalctl --vacuum-time{{2d}}`
```

Comment:
Maybe we missed a `=` :)
I think the command would consider it, but lets put an `=` in my opinion.
```suggestion
`journalctl --vacuum-time={{2d}}`
```

Reply:
No that is an error, good spot!